openedx/features/edly/utils.py

```diff
@@ -400,3 +404,18 @@ def clean_django_settings_override(django_settings_override):
 
     if validation_errors:
         raise ValidationError(validation_errors)
+
+
+def get_marketing_link(marketing_urls, name):
```

Comment:
We can use edx function `marketing_link`

Reply:
the `marketing_link` function from edx returns `MKTG_URLS` from the request site's configurations only, whereas we want `MKTG_URLS` for sites that are linked with the organizations of the user.